src/vote-subjects/entities/vote-subject.entity.ts

```diff
@@ -9,29 +9,29 @@ import {
 @Entity()
 export class VoteSubject {
   @PrimaryGeneratedColumn()
```

Comment:
```suggestion
  @PrimaryGeneratedColumn('uuid')
```
@sua-no `'uuid'` 옵션을 주지 않아도 uuid로 생성되는 건가요?

Reply:
uuid추가하겠습니다!